mex/extractors/consent_mailer/main.py

```diff
@@ -11,8 +20,36 @@
 def all_ldap_persons_for_mailer(
     extracted_primary_source_ldap: ExtractedPrimarySource,
 ) -> list[MergedPerson]:
-    """Get all LDAP persons."""
-    return extract_ldap_persons(extracted_primary_source_ldap.stableTargetId)
+    """Get LDAP persons without consent for mailer ."""
+    ldap_merged_persons = extract_ldap_persons(
+        extracted_primary_source_ldap.stableTargetId
+    )
+    merged_consents = extract_consents_for_persons(ldap_merged_persons)
+    return filter_persons_without_consent(ldap_merged_persons, merged_consents)
+
+
+@asset(group_name="consent_mailer")
+def send_consent_emails_to_persons(
+    all_ldap_persons_for_mailer: list[MergedPerson],
+) -> None:
+    """Send consent emails to the given persons.
+
+    Args:
+        all_ldap_persons_for_mailer (list[MergedPerson]): The list of persons that gets
```

Comment:
```suggestion
        all_ldap_persons_for_mailer: The list of persons that gets
```

no need to duplicate the typing in the docstring